src/i18n/strings/pl.json

```diff
@@ -567,6 +567,8 @@
     "Logs sent": "Zapis rozmów wysłany",
     "Back": "Powrót",
     "Reply": "Odpowiedz",
+    "Collapse quotes | ⇧+click": "Skurcz cytaty | ⇧+click",
+    "Expand quotes | ⇧+click": "Rozszerz cytaty | ⇧+click",
```

Comment:
These strings cannot be edited here. It will cause merge conflicts with weblate - please make them in weblate instead.